### ISSUE TITLE
Disable rule S2760

### ISSUE THREAD

**Issue by tvsonar** [SonarSource-VisualStudio/sonarlint-roslyn-analyzer#419]
Rule S2760 raises many false positives for users, so we should probably disable it by default.

* http://stackoverflow.com/questions/32656179/sonarlint-s2760-possible-bug

```
if (_rnd.NextDouble() < Settings.RandomChance)
{
    MethodA();
}
if (_rnd.NextDouble() < Settings.RandomChance)
{
    MethodB();
}
```

links to https://github.com/SonarSource/sonarlint-vs/issues/383

**Comment by JranZu** [comment]
I would advise maybe breaking this up into 2 rules. 1 rule for statements without methods and another-one with statements with methods.

This rules causes a ton of false positives when dealing with methods and I would personally have that turned off. However, I REALLY like seeing it when the statement deals with fields as it's helped out a lot with both shortening code and finding an occasional bug.

**Comment by njannink** [comment]
Even properties and fields may have changed between 2 calls (race conditions etc). I would just disable the whole rule, because it depends too much on the context if it is a positive or not.

```
if (instance == null)
{
    lock (lockObj)
    {
        if (instance == null)
        {
             instance = new Foo();
        }
    }
}
```

**Comment by tvsonar** [comment]
@njannink you are right that even properties and fields can change between the two checks. But I'm not sure why your example is relevant here. The rule checks consequtive `if` statements and not nested ones. However I see that in case of a multi-threaded application one thread can change a field, while the other thread is just between the consequtive `if`s.
